src/reference/asciidoc/index.adoc

```diff
@@ -41,3 +41,6 @@ include::si-amqp.adoc[]
 In addition to this reference documentation, there exist a number of other resources that may help you learn about AMQP.
 
 include::further-reading.adoc[]
+
+[appendix]
+include::appendix.adoc[]
```

Comment:
OK! 
Never mind.
But looks strange that the previous page with code isn't full because the next code block is so big that it is moved to the next page makes this What's New a big ugly.
I can play with this code block to move it to the previous page back.